docs/content/1-introduction.md

```diff
@@ -0,0 +1,185 @@
+
+`new`
+
+An invoice starts in the `new` state. Payments made to this invoice have a 20 minute window to be confirmed on the blockchain.
+
+`paid`
+
+After a payment was detected on the blockchain, an invoice is marked as `paid`. This is a transition state and the invoice will become either `expired` or `complete`.
+
+`complete`
+
+If the full payment made to an invoice has been confirmed on the blockchain during the confirmation window, the invoice is marked as `complete`. The merchant will be credited the invoice amount on the next payout.
+
+`expired`
+
+An invoice is marked as `expired` if the payment wasn't made in full or confirmed on the blockchain in time. All payments received will be automatically refunded.
+
+<b>See also:</b>
+
+* <a href="#invoice-webhooks">Invoice webhooks</a>
+
+## Errors
+
+The Nash Link API uses the following error codes:
+
+
+Error Code | Meaning
+---------- | -------
+400 | Bad Request -- Your request sucks
+401 | Unauthorized -- Your API key is incorrect
+404 | Not Found -- The specified resource could not be found
+500 | Internal Server Error -- We had a problem with our server – try again later
+503 | Service Unavailable -- We’re temporarially offline for maintanance – try again later
+
+## Activating API Access
+
+The merchant must obtain an API key from the Nash link web app to get access to the API calls.  
+
+The API generation/revoke actions are accessible at https://link.nash.io/developers/.  
+  
+A merchant can create multiple keys for use with different e-commerce stores or API functions.  
+
+Once an API key has been created, Nash link will use this API key to authenticate your API connections.  
+
+The merchant’s API key must remain private and should never be visible on any client-facing code.
+
+Should it ever be compromised, the merchant can generate a new key in their BitPay account.  
+
+## Our SDKs
+
+We provide well-maintained SDKs for several programming languages. These are a slim layer to simplify API access (in particular, request signing):
+
+* <a href="https://github.com/nash-io/nashlink-plugins-and-sdks/tree/main/sdk-node-typescript" target="_blank">Node.js / TypeScript</a>
+* <a href="https://github.com/nash-io/nashlink-plugins-and-sdks/tree/main/sdk-php" target="_blank">PHP</a>
+* <a href="https://github.com/nash-io/nashlink-plugins-and-sdks/tree/main/sdk-python" target="_blank">Python</a>
+
+<a href="#sdks">Read more</a>
+
+## Rest API Access
+
+Nash link provides a standards-based REST interface which enables application developers to interact in a powerful, yet secure way with their Nash link account.  
+
+Our REST API is highly compatible with <a href="https://bitpay.com/api" target="_blank">BitPay</a> and <a href="https://btcpayserver.org/" target="_blank">BTCPay</a>. It allows developers to interact with Nash Link securely for creating invoices as a merchant and retrieving invoices as a user.
```

Comment:
could you please remove the links :sweat_smile: ?

Reply:
done